packaging/rpm/make-airgap-rpm.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+imageStrore="/var/lib/microshift/airgap-images"
```

Comment:
nit :-) 
```suggestion
imageStore="/var/lib/microshift/airgap-images"
```

Reply:
It'd be great to remove all reference to "airgap(ped)" as this this implies more than just disconnected from network (Faraday cages, separate power grid, etc.).

Simply `/var/lib/microshift/images`?